.travis/common.sh

```diff
@@ -52,6 +52,7 @@ export GIT_SSL_NO_VERIFY=1
 export GITREV="$(git describe --long 2>/dev/null || echo "unknown")"
 export CONDA_BUILD_ARGS=$PACKAGE
 export CONDA_OUT="$(conda render --output $CONDA_BUILD_ARGS 2> /dev/null | grep conda-bld | grep tar.bz2 | sed -e's/-[0-9]\+\.tar/*.tar/' -e's/-git//' | tr '\n' ';')"
+export UHDM_INTEGRATION_REV="$(git ls-remote https://github.com/alainmarcel/uhdm-integration.git HEAD | awk '{ print $1}')"
```

Comment:
Actually, why do you need this?

Reply:
This is to make sure we don't have any race conditions in package compatibility, for example uhdm surelog build starts -> in the meantime somebody upgrades the submodules in uhdm integration -> only then uhdm verilator build starts with different submodules. Checking the revision once and using it for all packages ensures this won't happen.

Reply:
I don't think you have solved the problem you think you have solved here...  Give me a while to think about it....

Reply:
@tgorochowik - Do you know what the state of `conda build prepare` that @PiotrZierhoffer was managing? That was designed to solve exactly this problem by separating the "generating / freezing the conda recipe" from the build / test / etc steps.

Reply:
@tgorochowik @tjurtsch  @mithro This is kinda unrelated to conda-build-prepare.

The problem here is that we can have different revisions for different uhdm packages - Travis will not preserve the value across packages. They are built in different VMs.

Conda-build-prepare will help you debug/alter the code before building, but this is a totally different scope.

I think that you can share the git versions between packages in an external storage, no other way for that in Travis.

Reply:
I added a review where I wanted to add a comment. It's below